### DISCUSSION TITLE
picture of car and pictures with status of the car?

### DISCUSSION THREAD

**Discussion by togi99** [tillsteinbach/WeConnect-mqtt#240] [Q&A]
just started to play a little bit with WeConnect-mqtt.
Based on what I have read it should be possible to fetch pictures via MQTT.
Am I right?
How can this be achived? I have set parameter  --picture-format png.
But it seems my MQTT Broker does not get them
![grafik](https://github.com/tillsteinbach/WeConnect-mqtt/assets/140516669/c4dfe9ad-496e-4ebf-82fd-7ec34465a22a)

**Comment by kr-moller** [comment]
The pictures are under this topic:
weconnect/vehicles/XXXXXXXX/pictures

And I think you need both params:
--pictures --picture-format png
